Mathlib/Lean/Meta/Basic.lean

```diff
@@ -52,3 +53,18 @@ def Lean.Meta.forallMetaTelescopeReducingUntilDefEq
     bis := bis ++ bs
     out := tp
   return (mvs, bis, out)
+
+/-- Determines whether two expressions are definitionally equal to each other without any mvar
+    assignments. -/
+@[inline]
+def Lean.Meta.pureIsDefEq (e₁ e₂ : Expr) : MetaM Bool :=
+  withNewMCtxDepth <| isDefEq e₁ e₂
+
+/-- Similar to `mkAppM n #[lhs, rhs]`, but handles `Eq` and `Iff` more efficiently. -/
```

Comment:
We need to know from the documentation how it's similar, so here's a suggestion.

```suggestion
/-- `mkRel n lhs rhs` is `mkAppM n #[lhs, rhs]`, but with optimizations for `Eq` and `Iff`. -/
```